forui/test/src/widgets/accordion/accordion_controller_test.dart

```diff
@@ -0,0 +1,258 @@
+import 'package:flutter/animation.dart';
+import 'package:flutter/widgets.dart';
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/annotations.dart';
+import 'package:mockito/mockito.dart';
+
+import 'package:forui/forui.dart';
+import 'accordion_controller_test.mocks.dart';
+
+@GenerateNiceMocks([MockSpec<AnimationController>()])
+@GenerateNiceMocks([MockSpec<Animation>()])
+void main() {
+  group('FAccordionController', () {
```

Comment:
Just curious, but can the tests be simplified to use only 2 animation controllers/animations? Are there any test cases that explicitly require more than 2? Using 4 seems a little overkill.